frontend/src/components/pages/AdminPage.tsx

```diff
@@ -164,7 +199,30 @@ const AdminPage = (): React.ReactElement => {
                   </VStack>
                 </TabPanel>
                 <TabPanel>
-                  <AdminUserTable adminUsers={admins} />
+                  <VStack pt={4} spacing={6}>
+                    <HStack width="100%">
+                      <InputGroup width="95%">
+                        <Input
+                          borderRadius="6px"
+                          borderColor="grey.100"
+                          backgroundColor="grey.100"
+                          onChange={(e) => setSearch(e.target.value)}
+                          value={search}
+                          placeholder="Search bar"
+                        />
+                        <InputRightElement pointerEvents="none" h="full">
+                          <SearchOutlineIcon />
+                        </InputRightElement>
+                      </InputGroup>
+                      <SortTablePopover OrderingSets={OrderingSets} />
+                    </HStack>
+                    {search && (
+                      <Text fontSize="16px" color="grey.300" width="100%">
+                        Showing {admins.length} results for &quot;{search}&quot;
```

Comment:
```suggestion
                        Showing {teachers.length} results for &quot;{search}&quot;
```
double check that this returns the right values